docker/docs/zts-setup.md

```diff
@@ -0,0 +1,352 @@
+
+<a id="markdown-setup-zts-in-production-env" name="setup-zts-in-production-env"></a>
+# Setup ZTS in production env.
+
+<!-- TOC -->
+
+- [Setup ZTS in production env.](#setup-zts-in-production-env)
+    - [Prerequisites](#prerequisites)
+    - [Target](#target)
+    - [Steps](#steps)
+        - [1. update your passwords](#1-update-your-passwords)
+        - [2. get a server certificate for ZTS](#2-get-a-server-certificate-for-zts)
+        - [3. create ZTS key pairs for signing Athenz token](#3-create-zts-key-pairs-for-signing-athenz-token)
+        - [4. create trust store containing all the trusted CAs](#4-create-trust-store-containing-all-the-trusted-cas)
+        - [5. create key store containing the ZTS server certificate](#5-create-key-store-containing-the-zts-server-certificate)
+        - [6. set up for certificate signing](#6-set-up-for-certificate-signing)
+            - [HttpCertSigner](#httpcertsigner)
+            - [KeyStoreCertSigner](#keystorecertsigner)
+        - [7. set up for ZMS connection](#7-set-up-for-zms-connection)
+            - [Note](#note)
+        - [8. summary](#8-summary)
+        - [9. register ZTS service to Athenz](#9-register-zts-service-to-athenz)
+        - [10. create athenz.conf](#10-create-athenzconf)
+    - [Deploy ZTS](#deploy-zts)
+        - [Debug ZTS](#debug-zts)
+    - [Appendix](#appendix)
+        - [HttpCertSigner Details](#httpcertsigner-details)
+
+<!-- /TOC -->
+
+
+<a id="markdown-prerequisites" name="prerequisites"></a>
+## Prerequisites
+
+1. Env. setup done. ([env.sh](../env.sh))
+1. Bootstrap setup done. ([Done step 1, 2, 3, 4](./Athenz-bootstrap.md#bootstrap-steps))
+1. ZMS is running. ([zms-setup](./zms-setup.md))
+1. All the setup commands below are expected to run inside [athenz-setup-env](../setup-scripts/Dockerfile) container.
+```bash
+docker run --rm -it \
```

Comment:
```suggestion
BASE_DIR="`git rev-parse --show-toplevel`"
docker run --rm -it \
```

Reply:
no change

---

User has to set `DOCKER_NETWORK` in the command.
As stated in the prerequisites, user should run the env.sh already and set both `BASE_DIR` and `DOCKER_NETWORK` before running the command.